package.json

```diff
@@ -52,12 +52,14 @@
     "material-ui-search-bar": "^1.0.0-beta.13",
     "notistack": "https://github.com/gnosis/notistack.git#v0.9.4",
     "optimize-css-assets-webpack-plugin": "5.0.3",
+    "polish": "^0.2.3",
     "qrcode.react": "1.0.0",
     "react": "16.12.0",
     "react-dom": "16.12.0",
     "react-final-form": "6.3.3",
     "react-final-form-listeners": "^1.0.2",
     "react-hot-loader": "4.12.18",
+    "react-intercom": "^1.0.15",
```

Comment:
Why are we using this instead of official intercom script? This package was updated in October 2018 and there are already warnings that it'll be incompatible with the next version of react

Reply:
updated, check it now